README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+* Static variables
+* View Lifecycle (if available)
+* Public
+* Private
+* Extensions
+
+When implementing a protocol you should create an extension of your class that lives in the same file to separate the core logic of your class and your protocol implementation.
+
+#### Usage of MARK ####
+
+To help organizing your files you may want to use pragma marks to clearly separate your functions, properties, extensions.
```

Comment:
I think that's fine to have it optional, it's up to the team to decide to use them, I find them useful and you don't, but it's good having a rule for those who want to use them.